src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -6,27 +6,36 @@
     @close="resolve"
```

Comment:
Please use new modal design by adding `from-bottom` prop.
```suggestion
    from-bottom
    @close="resolve"
```